sass/theme.scss

```diff
@@ -1,14 +1,59 @@
+@import 'variables';
+
 * {
   font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
   outline: none;
   box-sizing: border-box;
 }
 
 body {
-  background: rgb(235, 235, 235);
   display: flex;
   margin: 0;
   padding: 0;
+
+  @each $theme,
+  $colors in $themes {
+    &.theme-#{$theme} {
+      background-color: map-get($colors, 'base-color');
+
+      button {
+        background-color: transparent;
+        color: map-get($colors, 'border-color');
+      }
+
+      .sidebar {
+        background-color: map-get($colors, 'accent-color');
+      }
+
+      textarea {
+        background-color: map-get($colors, 'object-color');
+        color: map-get($colors, 'accent-color');
+      }
+
+      .close {
+        color: map-get($colors, 'border-color');
+      }
+
+      .sticker-body {
+        background-color: map-get($colors, 'object-color');
+      }
+
+      .info {
+        background-color: map-get($colors, 'accent-color');
+        color: map-get($colors, 'border-color');
+      }
+
+      .footer {
+        background-color: map-get($colors, 'object-color');
+        border-bottom: map-get($colors, solid 1px 'border-color');
+      }
+
+      .sticker-container {
+        background-color: map-get($colors, 'base-color');
+      }
+    }
+  }
+
 }
 
 textarea {
```

Comment:
Remove all the color definitions outside the theme loop (below this line).
Colors should be handled only by theme definitions.